### ISSUE TITLE
🐛 BUG: Importing css-files using @import in scss-files causes the generated css-file to be omitted on build

### ISSUE THREAD

**Issue by dnbo** [snowpackjs/astro#1515]
### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

Create a scss-file in a folder and import it in an Astro-component like (importing the generated css file as specified in docs)

`<link rel="stylesheet" href={Astro.resolve('../assets/styles/app.css')} type="text/css">`

In the scss-file, import a css-file using `@import`-syntax like

`@import '../css/app.css';`.

When running `astro dev` everything works as expected and `_astro/src/assets/styles/app.css` is included, but when running `astro build` the file is omitted.

### Steps to Reproduce

 # Issue
`npm run dev` works fine while `npm run build` wont include the `/styles/app.css` in the output.

 ## Folder structure that causes the bug

If styles are arranged as below bug appears

```
/
├── public/
│   ├── robots.txt
│   └── favicon.ico
├── src/
│   ├── assets/
│   │   └── css
|   |   |   └── app.css
|   |   └── styles
|   |       └── app.scss        (this file @import's the ../css/app.css file)
```

The css is then imported in the `index.astro` like below

`<link rel="stylesheet" href={Astro.resolve('../assets/styles/app.css')} type="text/css">`

### Link to Minimal Reproducible Example (Optional)

https://github.com/dnbo/astro-scss-bug-repro

**Comment by natemoo-re** [comment]
This should be fixed in the upcoming `astro@next` release!

Here is a working demo using `astro dev` on `astro@0.21.0-next.2`
https://stackblitz.com/edit/github-xmjcr1?file=src%2Fpages%2Findex.astro

We're still finishing up the `astro build` functionality to be released in `0.21.0-next.3`. PR for that should be coming soon.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
